### PR TITLE
Submitting the Corporate BS Generator

### PR DESCRIPTION
**Pull request by ryantaylor16**
# Description
I am submitting the Corporate BS Generator

# Copilot

### Summary
🏢💬🎲

This pull request adds a new app called Corporate BS Gen, which displays a random phrase of corporate jargon on the Tidbyt device. The app consists of a Starlark code file `corporate_bs_gen.star` and a manifest file `manifest.yaml`.

> _`BS Gen` app code_
> _Fetches jargon from the web_
> _Displays on Tidbyt_

### Walkthrough
* Create and configure a new app called Corporate BS Gen that displays random corporate jargon phrases on the Tidbyt device ([link](https://github.com/tidbyt/community/pull/1774/files?diff=unified&w=0#diff-a61a528f80d1fb66d0e333196eb70f1d5c92ec41f4110ad1dd56b255bbc4d3fcR1-R42), [link](https://github.com/tidbyt/community/pull/1774/files?diff=unified&w=0#diff-726039d5f93a7e173d1f9e20b6671c9e564db65b2091ad8e9269d156f222c49aR1-R8))
  * Use the `http` module to fetch a phrase from the https://corporatebs-generator.sameerkumar.website/ API ([link](https://github.com/tidbyt/community/pull/1774/files?diff=unified&w=0#diff-a61a528f80d1fb66d0e333196eb70f1d5c92ec41f4110ad1dd56b255bbc4d3fcR1-R42))
  * Use the `render` module to display the phrase as a marquee text with a logo and a title ([link](https://github.com/tidbyt/community/pull/1774/files?diff=unified&w=0#diff-a61a528f80d1fb66d0e333196eb70f1d5c92ec41f4110ad1dd56b255bbc4d3fcR1-R42))
  * Use the `base64` module to encode the logo image as a string and store it in the `logo` variable ([link](https://github.com/tidbyt/community/pull/1774/files?diff=unified&w=0#diff-a61a528f80d1fb66d0e333196eb70f1d5c92ec41f4110ad1dd56b255bbc4d3fcR1-R42))
  * Define the app metadata and configuration in the `manifest.yaml` file, including the app id, name, summary, description, author, file name, and package name ([link](https://github.com/tidbyt/community/pull/1774/files?diff=unified&w=0#diff-726039d5f93a7e173d1f9e20b6671c9e564db65b2091ad8e9269d156f222c49aR1-R8))

### CODE REVIEW

**Comment by ryantaylor16** [comment]
> 

I have read the CLA Document and I hereby sign the CLA

**Comment by ryantaylor16** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/tidbyt/community/blob/main/docs/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> _**I have read the CLA Document and I hereby sign the CLA**_
> 
> You can retrigger this bot by commenting **recheck** in this Pull Request

recheck

**Comment by ryantaylor16** [comment]
**recheck**

**Comment by ryantaylor16** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by ryantaylor16** [comment]
recheck

File: apps/corporatebsgen/corporate_bs_gen.star [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+""")
+
+def main():
+    phrase = cache.get("corporate_bs")
+    if phrase != None:
+        print("Cache Hit!")
+    else:
+        print("Cache Miss!")
+        rep = http.get(CORPORATE_BS)
```
Comment (matslina) [inline apps/corporatebsgen/corporate_bs_gen.star:17]: You can now pass `ttl_seconds` straight to `http.get()` to have the http module handle caching for you. This simplifies the code a bit, and also future proofs against (possible) coming changes to how cache module is scoped.

**Review by matslina — APPROVED** [review]
Thanks for the PR. Would be great if you could address the comment about caching in a later PR.

**Pull request merged**